### PR TITLE
[MIG] web_environment_ribbon: Migration to 13.0

### PR DESCRIPTION
**Pull request by lembregtse**
Proposed migration of web_environment_ribbon to 13.0.

Added trivial test case to get some test coverage with this module.
Removed jquery dependency since it is included by default and no longer works as a requirement.

### CODE REVIEW

File: web_environment_ribbon/tests/test_environment_ribbon_data.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    def setUpClass(cls):
+        super(TestEnvironmentRibbonData, cls).setUpClass()
+        cls.env['ir.config_parameter'].set_param(
+            'ribbon.name', 'Test Ribbon {db_name}')
+        cls.env['ir.config_parameter'].set_param('ribbon.color', '#000000')
+        cls.env['ir.config_parameter'].set_param(
+            'ribbon.background.color', '#FFFFFF')
+
+    def test_environment_ribbon(self):
+        """We have to test in a tricky way, as res.users doesn't allow a
```
Comment (schout-it) [inline web_environment_ribbon/tests/test_environment_ribbon_data.py]: copy - pasta?

**Review by schout-it — COMMENTED** [review]
comment in test seems strange, copy / paste problem.
Rest LGTM

**Review by schout-it — APPROVED** [review]
lgtm

**Review by emagdalenaC2i — APPROVED** [review]
Code review

**Pull request merged**